jsonrpc/handlerT.go

```diff
@@ -0,0 +1,160 @@
+package jsonrpc
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"regexp"
+	"runtime/pprof"
+	"runtime/trace"
+	"strings"
+	"sync"
+
+	"github.com/hashicorp/go-bexpr"
+)
+
+// HandlerT implements the debugging API.
+// Do not create values of this type, use the one
+// in the Handler variable instead.
+type HandlerT struct {
```

Comment:
Let's call it:
```suggestion
type DebugHandler struct {
```

Don't forget to update the doc of the struct.